demo/handoff/vat-handoff.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019 Agoric, under Apache License 2.0
+
+import harden from '@agoric/harden';
+
+import { makeHandoffService } from './handoff';
```

Comment:
What was the decision-making process for choosing to create a vat for the handoff service versus using it as a library in bootstrap?

Reply:
I don't understand the second option. The handoff service should be in a separate vat from the users. For the demo, I'm expecting it to be in a separate vat that's included at startup.  What would it look like to use it as a library?

Reply:
Having it's own vat separate from the bootstrap vat might be the right option. I was just wondering because the bootstrap vat is already a separate vat from the users, so there might be further reasons that I didn't understand.